src/Compilers/Core/Portable/Compilation/Extensions.cs

```diff
@@ -135,6 +146,46 @@ public static ISymbol GetDeclaredSymbol(this SemanticModel semanticModel, Syntax
             return semanticModel.GetDeclaredSymbolForNode(declaration, cancellationToken);
         }
 
+        public static TypeInfo GetTypeWithNullability(this SemanticModel semanticModel, IDiscardSymbol discard, CancellationToken ct = default)
+        {
+            throw new NotImplementedException();
+        }
+
+        public static TypeInfo GetTypeWithNullability(this SemanticModel semanticModel, IEventSymbol eventSymbol, CancellationToken ct = default)
+        {
+            throw new NotImplementedException();
+        }
+
+        public static TypeInfo GetTypeWithNullability(this SemanticModel semanticModel, IFieldSymbol field, CancellationToken ct = default)
+        {
+            throw new NotImplementedException();
+        }
```

Comment:
what if you don't have a semantic model... feels super weird that this would be at the semantic model level.

Consider any and all IDE features that do any sort of generation.  Previously there was no need to pass along any sort of compilation.  They could just use symbols.

Note: this affects *public* APIs we've shipped.  These APIs would now seemingly become 'wrong' for any cases involving nullability.  taht concerns me greatly.